feature/experimental/p4rt/ate_tests/base_p4rt/base_p4rt_test.go

```diff
@@ -128,6 +127,9 @@ func configureDeviceIDs(ctx context.Context, t *testing.T, dut *ondatra.DUTDevic
 		c.IntegratedCircuit.NodeId = ygot.Uint64(deviceIDs[idx])
 		gnmi.Replace(t, dut, gnmi.OC().Component(nodes[p]).Config(), &c)
 	}
+	if nodes["port1"] == nodes["port2"] {
```

Comment:
do we need to gate on the existence of `nodes["port1"]` and `nodes["port2"]`? (i.e such key exists.)

Reply:
Not in this particular case since this check happens earlier in this function call on L120 https://github.com/openconfig/featureprofiles/blob/65988b7cc3a8cb75173f214a3ab7dbb5255868cd/feature/experimental/p4rt/ate_tests/base_p4rt/base_p4rt_test.go#L119-L122